mmdet/models/losses/kd_loss.py

```diff
@@ -0,0 +1,93 @@
+import mmcv
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import LOSSES
+from .utils import weighted_loss
+
+
+@mmcv.jit(derivate=True, coderize=True)
+@weighted_loss
+def knowledge_distillation_loss(pred, soft_label, T, detach_target=True):
+    r"""Loss function of `Distilling the Knowledge in a Neural Network.
+    <https://arxiv.org/abs/1503.02531>`_.
+
+    Args:
+        pred (torch.Tensor): Predicted general distribution of bounding boxes
```

Comment:
pred (Tensor): Predicted logits with shape (N, n + 1). 
soft_label (Tensor): Target logits with shape (N, N + 1). 
T (float): Temperature for distillation.